scrapy/extensions/feedexport.py

```diff
@@ -180,14 +180,16 @@ def _store_in_thread(self, file):
 
 
 class _FeedSlot:
-    def __init__(self, file, exporter, storage, uri, format, store_empty):
+    def __init__(self, file, exporter, storage, uri, format, store_empty, batch_id, template_uri):
```

Comment:
I think `uri_template` would be more accurate, it’s the template of a URI, not the URI of a template.